src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/RestClients/ResourceManagerRestClientBase.cs

```diff
@@ -219,27 +220,39 @@ public ResourceManagerRestClientBase(Components.HttpClientHelper httpClientHelpe
         {
             using (var httpClient = this.httpClientHelper.CreateHttpClient())
             {
-                var response = await httpClient
-                    .SendAsync(request: request, cancellationToken: cancellationToken)
-                    .ConfigureAwait(continueOnCapturedContext: false);
-
-                if (!response.StatusCode.IsSuccessfulRequest())
+                try
                 {
-                    var errorResponse = await ResourceManagerRestClientBase
-                        .TryReadErrorResponseMessage(response, rewindContentStream: true)
+                    var response = await httpClient
+                        .SendAsync(request: request, cancellationToken: cancellationToken)
```

Comment:
sounds good

---
In reply to: [66568621](https://github.com/Azure/azure-powershell/pull/2412#discussion_r66568621) [](ancestors = 66568621)